interview-questions.md

```diff
@@ -9,41 +9,41 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: in JavaScript, 'this' refers to the object it is contained within/being called from. 
 
-  Researched answer:
+  Researched answer: 
 
 
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: Yarn is an application that takes your React application and components and balls them all up together like a ball of yarn.
 
-  Researched answer:
+  Researched answer: Yarn is a package manager which allows you to use and share code with other developers, yarn adds a package file and a yarn.lock file to your react app.
```

Comment:
WHat else about it? what did you learn from research? Who made it? Why is it different? What makes it work?